CMakeLists.txt

```diff
@@ -114,6 +114,9 @@ if (NOT OPCUASERVER)
 endif()
 target_link_libraries(${PROJECT_NAME} ${OPCUASERVER} ${OPCUACORE} ${OPCUAPROTOCOL})
 
+find_package(Boost 1.53.0 COMPONENTS thread REQUIRED)
+target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES})
```

Comment:
If its required with latest freeopcua library. Then same fix applies to `foglamp-south-opcua`?

Reply:
It is related to the opcuaserver library that is only used in the north. After checking the opcua south plugin with the latest freeopcuv there is no problem.